README.md

```diff
@@ -217,6 +217,7 @@ Find the **Protocol Buffer Definitions** in the `/pb/` directory.
 
 ## Demos featuring Online Boutique
 
+- [Honeycomb.io](https://github.com/honeycombio)
```

Comment:
This link leads to a GitHub org page, maybe https://github.com/honeycombio/opentelemetry-demo-webstore?

Reply:
Yeah that's what I meant to use